src/hooks/useNodeJsContributorsApi.tsx

```diff
@@ -1,20 +1,29 @@
+/* eslint-disable camelcase */
 import { useEffect, useState } from 'react';
+import fetch from 'node-fetch';
```

Comment:
```suggestion
```

Reply:
If we do this change, the jest unit test will occurred an error ("fetch is not defined").
If you have an solution tell me.

Reply:
Refer the old test file. I think you need to use the fetchMock